projects/dav1d/Dockerfile

```diff
@@ -20,7 +20,9 @@ MAINTAINER janne-vlc@jannau.net
 ADD bionic.list /etc/apt/sources.list.d/bionic.list
 ADD nasm_apt.pin /etc/apt/preferences
 
-RUN apt-get update && apt-get install -y curl python3-pip nasm && \
+RUN dpkg --add-architecture i386 && \
+    apt-get update && \
+    apt-get install -y curl libstdc++-5-dev libstdc++-5-dev:i386 python3-pip nasm && \
```

Comment:
Is `libstdc++-5-dev ` required for building the fuzzing engine or for for linking the final fuzzing binary?

Reply:
I see it is for some reason required after adding i386 support